tests/test_other.py

```diff
@@ -12226,3 +12226,54 @@ def test_rust_gxx_personality_v0(self):
         }
       }
     ''', assert_returncode=NON_ZERO)
+
+  def test_safari_bigint64array_shim(self):
+    bigint64array = Path(path_from_root("src/polyfill/bigint64array.js")).read_text()
+    test_code = read_file(test_file('test_safari_bigint64array_shim.js'))
+    bigint_list = [
+      0,
+      1,
+      -1,
+      5,
+      (1 << 64),
+      (1 << 64) - 1,
+      (1 << 64) + 1,
+      (1 << 63),
+      (1 << 63) - 1,
+      (1 << 63) + 1,
+    ]
+    bigint_list_strs = [str(x) for x in bigint_list]
+
+    bigint_list_unsigned = [x % (1 << 64) for x in bigint_list]
+    bigint_list_signed = [
+      x if x < 0 else (x % (1 << 64)) - 2 * (x & (1 << 63)) for x in bigint_list
+    ]
+    bigint_list_unsigned_n = [f"{x}n" for x in bigint_list_unsigned]
+    bigint_list_signed_n = [f"{x}n" for x in bigint_list_signed]
+
+    bigint64array = "".join(bigint64array.splitlines(keepends=True)[3:])
+
+    create_file(
+      "test.js",
```

Comment:
Can you use single quotes here in the python test code (where possible).

Reply:
What about in JavaScript?

Reply:
yes we prefer single quotes there too IIUC.